jwst/pipeline/tests/data/cfgs/outlier_detection.cfg

```diff
@@ -14,4 +14,4 @@ scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
 resample_data = True
-good_bits = 4
+good_bits = 'JUMP_DET'
```

Comment:
What are all these config files doing in `jwst/pipeline/tests/data/cfgs/`?  This seems like a recipe for being out-of-date?

Reply:
yep. should be removed once all configs have been removed/simplified.